core/src/main/java/org/elasticsearch/cluster/action/shard/ShardStateAction.java

```diff
@@ -207,9 +207,11 @@ public void onTimeout(TimeValue timeout) {
 
         @Override
         public void messageReceived(ShardEntry request, TransportChannel channel) throws Exception {
-            logger.warn((Supplier<?>) () -> new ParameterizedMessage("{} received shard failed for {}", request.shardId, request), request.failure);
+            logger.warn((Supplier<?>) () -> new ParameterizedMessage("{} received shard failed for {}",
```

Comment:
This is just a log message. I think we want all the details.

Reply:
The missing detail is just `request.failure` which seems to be logged anyway. This change just stops it logging that twice, I think.

Reply:
I missed the logging of the exception. All good then.